File: 2942.find-words-containing-character.py
#
# @lc app=leetcode id=2942 lang=python
#
# [2942] Find Words Containing Character
#

# @lc code=start
class Solution(object):
    def findWordsContaining(self, words, x):
        """
        :type words: List[str]
        :type x: str
        :rtype: List[int]
        """
        count = []
        for i in range(len(words)):
            if x in words[i]:
                count.append(i)
        return count

        
        
# @lc code=end

File: 2942.find-words-containing-character.java
/*
 * @lc app=leetcode id=2942 lang=java
 *
 * [2942] Find Words Containing Character
 */

// @lc code=start
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> ret = new ArrayList<Integer>();
        for (int i = 0;i<words.length;i++) {
            if (words[i].contains(String.valueOf(x))) {
                ret.add(i);
            }
        }
        return ret;
    }
}
// @lc code=end

